### PR TITLE
Replace javascript object with class

### PR DESCRIPTION
**Pull request by misikira**
1. a collection that keeps a list of books
2. a function to add a new book to the collection, with title and author.
3. a function to remove a book from the collection
4. Display all books saved in the collection at the top part of the page.
5. data is preserved in the browser's memory by using localStorage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,10 @@
 
 - [📝 License](#license)
 
+### Key Features <a name="key-features"></a>
+- **[JS Class Practices]**
+- **[Medium-fidelity Wireframes]**
+
```
Comment (mikethreels) [inline README.md:30]: - [ ] good job using the readme template from Microverse, it seems like you duplicated it by accident.
- [ ] also the link to the MIT licence seems to be broken please make sure to fix it.

File: book.js [hunk 2]
```diff
@@ -1,58 +1,70 @@
-let books = [
-];
-
-const bookName = document.querySelector('.book');
-const authorName = document.querySelector('.author');
-const add = document.querySelector('.add');
+/* eslint max-classes-per-file: ["error", 2] */
```
Comment (mikethreels) [inline book.js:1]: - [ ] please avoid turning off the Linters they are used to make our code look cleaner and easier to read.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job implementing the classes

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @misikira 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Js classes used properly :\
* No linter errors 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**